python3-wiremapper.json

```diff
@@ -0,0 +1,44 @@
+{
+    "name": "python3-wiremapper",
+    "buildsystem": "simple",
+    "build-commands": [
+        "pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} wiremapper"
+    ],
+    "sources": [
+        {
+            "type": "file",
+            "url": "https://files.pythonhosted.org/packages/51/51/cf3bc0f9fef43bd55813fa9f8cc58f55ad3f95fc7f470b3a7b4c434fd79a/cobs-1.1.3.tar.gz",
+            "sha256": "ae3319b5704d9269d1ca60787d2e5b79322bb9f440215ea086b986ed8d2a8405"
+        },
+        {
+            "type": "file",
+            "url": "https://files.pythonhosted.org/packages/a6/e0/70a44c4385f2b33df82e518005aae16b5c1feaf082c73c0acebe3426fc0a/crc16-0.1.1.tar.gz",
+            "sha256": "c1f86aa0390f4baf07d2631b16b979580eae1d9a973a826ce45353a22ee8d396"
+        },
+        {
+            "type": "file",
+            "url": "https://files.pythonhosted.org/packages/52/e7/464079606a9cf97ad04936c52a5324d14dae36215f9319bf3faa46a7907d/scapy-2.4.3.tar.gz",
+            "sha256": "e2f8d11f6a941c14a789ae8b236b27bd634681f1b29b5e893861e284d234f6b0"
+        },
+        {
+            "type": "file",
+            "url": "https://files.pythonhosted.org/packages/62/7e/d4fb56a1695fa65da0c8d3071855fa5408447b913c58c01933c2f81a269a/dbus-python-1.2.16.tar.gz",
+            "sha256": "11238f1d86c995d8aed2e22f04a1e3779f0d70e587caffeab4857f3c662ed5a4"
+        },
+        {
```

Comment:
already in the runtime

Reply:
no it isn't

Reply:
just checked, indeed it isn't there anymore